packages/storybook/stories/va-table-uswds.stories.jsx

```diff
@@ -370,44 +362,53 @@ const sortData = [
     'Ninth',
     'August',
     'June 3, 1903',
-    "Lorem ipsum dolor sit,"
+    'Lorem ipsum dolor sit,',
   ],
   [
     '8.9',
     '1%',
     '$17,000',
     '3rd',
     'Second',
-    "February",
+    'February',
     'October 25, 1415',
-    "amet consectetur adipisicing elit."
+    'amet consectetur adipisicing elit.',
   ],
   [
     '-5',
     '60.01%',
     '$100,000',
-    "8th",
+    '8th',
     'Fifth',
-    "November",
+    'November',
     'December 10, 1621',
-    "Alias nam eum minima",
+    'Alias nam eum minima',
   ],
   [
     '99',
     '44%',
     '$1,100',
-    "5th",
+    '5th',
     'First',
-    "April",
+    'April',
     'September 30, 1885',
-    "delectus explicabo"
-  ]
-]
+    'delectus explicabo',
+  ],
+];
 
 export const Sortable = Template.bind(null);
 Sortable.args = {
-  'table-title': "This is a sortable table",
-  rows: sortData,
-  columns: sortColumns,
-  sortable: true
-}
\ No newline at end of file
+  'table-title': 'This is a sortable table',
+  'rows': sortData,
+  'columns': sortColumns,
+  'sortable': true,
+};
+
+export const Scrollable = Template.bind(null);
+Scrollable.args = {
+  'table-title': 'This is a scrollable table.',
+  'rows': data,
+  'columns': defaultColumns,
+  'scrollable': true,
```

Comment:
It looks like the [WithPagination Storybook variation](https://github.com/department-of-veterans-affairs/component-library/blob/de72c659f98046d6934cc27da43e65df3cada74c/packages/storybook/stories/va-table-uswds.stories.jsx#L332-L336) needs the `scrollable` prop too because it contains more content. I'm unable to scroll it now on [small screens](https://patch-bmt2-va-table-focusable-prop--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/story/uswds-va-table--with-pagination) (before it stacks).  Can you add that prop to [that story](https://github.com/department-of-veterans-affairs/component-library/blob/de72c659f98046d6934cc27da43e65df3cada74c/packages/storybook/stories/va-table-uswds.stories.jsx#L332-L336) too?

Reply:
@janechodance I noticed one more thing from the[ USWDS example](https://designsystem.digital.gov/components/table/#scrollable-table); the default behavior for scrollable tables is to also have `stacked` set to `false`. 

![Screenshot 2024-12-11 at 11 34 10 AM](https://github.com/user-attachments/assets/29d5e5a3-3375-4fe9-9290-f419b722034b)

Originally we were thinking that we'd let it stack but I'm going to tag @rsmithadhoc to get his thoughts about if we should add the `stacked: false` prop to our scrollable Storybook example so that it matches USWDS? 🤔 I think I'm fine either way but just looking for a sanity check.

Reply:
> @janechodance I noticed one more thing from the[ USWDS example](https://designsystem.digital.gov/components/table/#scrollable-table); the default behavior for scrollable tables is to also have `stacked` set to `false`.
> 
> Originally we were thinking that we'd let it stack on small screens but I'm going to tag @rsmithadhoc to get his thoughts about if we should add the `stacked: false` prop to our scrollable Storybook example so that it matches USWDS? 🤔 I think I'm fine either way but just looking for a sanity check.
> 
> ![Screenshot 2024-12-11 at 11 34 10 AM](https://private-user-images.githubusercontent.com/872479/394854539-29d5e5a3-3375-4fe9-9290-f419b722034b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMTU4OTksIm5iZiI6MTczNDAxNTU5OSwicGF0aCI6Ii84NzI0NzkvMzk0ODU0NTM5LTI5ZDVlNWEzLTMzNzUtNGZlOS05MjkwLWY0MTliNzIyMDM0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMlQxNDU5NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNzUzMTNmNzRmODgzYmFlOGI3YTk0YmQ1NzNhMTQ4NTI4NGIyZDJiNzIwYjY4Mzc0MTRjNDJmMzcyMWQ1MzgyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.24eQimOY8I314pY_gdvPfzzCl-a6yyI40h-VDg0KoWI)

@jamigibbs I think I initially said make it stacked by default when we first did the table, just to eliminate any work from teams. However, I'm not too tied to that idea, so I could also go either way. I think the prop would give teams all the options at their disposal, so it might be the most flexible option.

Reply:
@rsmithadhoc Thanks Ryan! Yes, I agree it would be good to see the flexibility with the options.